Library/Formula/libvbucket.rb

```diff
@@ -3,7 +3,7 @@
 class Libvbucket < Formula
   desc "Utility library providing mapping to virtual buckets"
   homepage 'http://couchbase.com/develop/c/current'
```

Comment:
It is mixed-content, but these are usually accepted. (should be fixed upstream eventually)